csaf_2.1/prose/edit/src/distributing.md

```diff
@@ -558,6 +558,19 @@ Each such folder MUST at least:
   }
 ```
 
+### Requirement 24: HTTP User-Agent
+
+Access to the files provided, both CSAF metadata and documents, MUST NOT be restricted by the value of HTTP User-Agent.
```

Comment:
I will try to provide an alernative version of this section soon (but not now) trying to mitigate my two concerns:

1. Use of `MUST NOT` instead of phrasing a `MUST` sentence.
2. High `MUST` density in sentences with "escape clauses" like
   > MUST be checked if ...

   and:

   > MUST be considered

In addition, we might follow the fashion (as with TLP) to use an alternative term to white-listing.

Reply:
Thank you for your comments. I addressed 1 and your addition in [df33caa](https://github.com/oasis-tcs/csaf/pull/742/commits/df33caa90cafe4eea5d306040a9c64c0236ea430).

Reply:
Unfortunately, I have no idea how to address 2 without loosing clarity.

Reply:
Addressed in [65dea4f](https://github.com/oasis-tcs/csaf/pull/742/commits/65dea4f650f19f4e5bd6599038c82e3e9fdf87b7)